### PR TITLE
CI: add issues-helper

### PR DESCRIPTION
**Pull request by xrkffgg**
I find you often label `awaiting-reply` to remind other to provide a recurrence demo.

❤️ I think this action will make it easy for you.

### introduce
- issue-labeled
    - When you label `awaiting-reply`, the github-bot will help you comment. Like: https://github.com/xrkffgg/bootstrap/issues/1
- issue-close-require
    - When issues label `awaiting-reply` inactive 7 days, GitHub Actions will close the issue and comment.

### CODE REVIEW

**Comment by GeoSot** [comment]
**/CC** @mdo @XhmikosR  the auto-close job seems handy 
(the auto-respond I think won't be always valid)

File: .github/workflows/issue-close-require.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  issue-close-require:
+    runs-on: ubuntu-latest
+    steps:
+      - name: awaiting reply
+        uses: actions-cool/issues-helper@v3
+        with:
+          actions: 'close-issues'
+          labels: 'awaiting-reply'
+          inactive-day: 7
```
Comment (XhmikosR) [inline .github/workflows/issue-close-require.yml:16]: Let's make this 14

File: .github/workflows/issue-labeled.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: awaiting reply
+        if: github.event.label.name == 'awaiting-reply'
+        uses: actions-cool/issues-helper@v3
+        with:
+          actions: 'create-comment'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          body: |
+            Hello @${{ github.event.issue.user.login }}. Bug reports must include a **live demo** of the issue. Per our [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md), please create a reduced test case via [CodePen](https://codepen.io/) or [JS Bin](https://jsbin.com/) and report back with your link, Bootstrap version, and specific browser and Operating System details. Thanks.
```
Comment (XhmikosR) [inline .github/workflows/issue-labeled.yml:18]: This is the template I'm using:

```markdown
Bug reports must include a **live demo** of the issue. Per our [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md), please create a reduced test case via [CodePen](https://codepen.io/) or [JS Bin](https://jsbin.com/) and report back with your link, Bootstrap version, and specific browser and Operating System details.
```

File: .github/workflows/issue-close-require.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: Issue Close Require
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  issue-close-require:
+    runs-on: ubuntu-latest
```
Comment (XhmikosR) [inline .github/workflows/issue-close-require.yml:9]: We should limit the workflow to our repo (similarly the other one too):

```yml
if: github.repository == 'twbs/bootstrap'
```

Comment (xrkffgg) [reply .github/workflows/issue-close-require.yml:9]: done

File: .github/workflows/issue-labeled.yml (hunk 2 above)
Comment (xrkffgg) [reply .github/workflows/issue-labeled.yml:18]: Yeah, I'm using your template. Before see https://github.com/twbs/bootstrap/issues/35708

**Comment by xrkffgg** [comment]
@XhmikosR HI,  what else needs to do about this pr?

**Comment by XhmikosR** [comment]
@patrickhlauke @mdo can you check for any grammar fixes please?

File: .github/workflows/issue-labeled.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: awaiting reply
+        if: github.event.label.name == 'awaiting-reply'
+        uses: actions-cool/issues-helper@v3
+        with:
+          actions: 'create-comment'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          body: |
+            Hello @${{ github.event.issue.user.login }}. Bug reports must include a **live demo** of the issue. Per our [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md), please create a reduced test case via [CodePen](https://codepen.io/) or [JS Bin](https://jsbin.com/) and report back with your link, Bootstrap version, and specific browser and Operating System details.
```
Comment (patrickhlauke) [inline .github/workflows/issue-labeled.yml:19]: just a minor one... i'd replace `via` with `on`

**Comment by xrkffgg** [comment]
@patrickhlauke Thx, done.

**Comment by XhmikosR** [comment]
Thanks for the contribution!

We'll give this a go and see :)

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**